### PR TITLE
Refactor contact card

### PR DESCRIPTION
**Pull request by ColinStClaire**
made population of contact card dynamic, obviously still pulling data from mock DB

### CODE REVIEW

File: TicketDesk/ngWebClient-StandAlone/src/app/ticket-submit/ticket-submit.component.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { Component, OnInit } from '@angular/core';
-import { Ticket, BLANK_TICKET } from '../models/data';
+import { Ticket, BLANK_TICKET } from '../models/ticket';
```
Comment (kixiQu) [inline TicketDesk/ngWebClient-StandAlone/src/app/ticket-submit/ticket-submit.component.ts:2]: Are we confident we've replaced all the old imports?

File: TicketDesk/ngWebClient-StandAlone/src/app/single-ticket-view/single-ticket-view.component.ts [hunk 2]
```diff
@@ -3,32 +3,36 @@ import { ActivityLogComponent } from '../activity-log/activity-log.component';
...
   constructor(private singleTicketService: SingleTicketService,
     private activatedRoute: ActivatedRoute) {
     this.activatedRoute.params.subscribe(params => {
       this.ticketId = Number(params['ticketID']);
     });
   }
 
+  ticketIsOpen(): boolean {
+    const status = this.ticketStatus.toLowerCase();
+    return status.indexOf('active') !== -1
```
Comment (kixiQu) [inline TicketDesk/ngWebClient-StandAlone/src/app/single-ticket-view/single-ticket-view.component.ts:29]: is this how people do this??????????????? it looks very intentional, but it's making me real uncomfortable

**Review by kixiQu — COMMENTED** [review]
The giant change to the libraries is making me uncomfortable. Do we know why that's so different?

**Review by kixiQu — CHANGES_REQUESTED** [review]
you need to add to your own .gitignore .DS_Store (Macs add this) and .vscode (your editor is adding this) and add a commit git `rm`-ing them (you might do the package-lock.json as well because while this is kosher to HAVE in version control, the fact that yours is SO different is concerning) but then we're good

**Review by kixiQu — DISMISSED** [review]

File: TicketDesk/ngWebClient-StandAlone/src/app/single-ticket-view/single-ticket-view.component.ts (hunk 2 above)
Comment (JoschaO) [reply TicketDesk/ngWebClient-StandAlone/src/app/single-ticket-view/single-ticket-view.component.ts:29]: Agreed, why not use something like .includes() instead? Index seems unnecessary amount of work.

**Review by JoschaO — DISMISSED** [review]
Besides the status function looks a little weird. 
Also concerned about imports.

**Review by JoschaO — APPROVED** [review]
Want to merge

**Pull request merged**